src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -59,6 +65,7 @@ class Lambda extends RequestStreamHandler {
       entitiesClient <- entitiesClientIO
       secretName = config.secretName
 
+      _ <- log(s"Searching for Source Ids ${folderRowsSortedByParentPath.map(_.name).mkString(",")}")
```

Comment:
Could these names be sensitive?

Reply:
I don't think so at this point for judgments and by the time we do get anything sensitive, we'll be encrypting the logs. At this point, it's just dumping stuff in there and seeing what's useful so we can always change our minds.